tests/plugins/subcommands/doctor/test_health_checks.py

```diff
@@ -67,6 +111,11 @@ def test_display_health_checks(env_ok: tuple[Path, str, str, str], verbose: bool
         {"CONDA_PREFIX": prefix},
         stack_callback=conda_tests_ctxt_mgmt_def_pol,
     ):
-        display_health_checks(prefix, verbose=verbose)
-        (prefix / bin_doctor).unlink()
-        display_health_checks(prefix, verbose=verbose)
+        display_health_checks(
```

Comment:
Can you please add some assertions to this test using the `capsys` fixture available from pytest? (https://docs.pytest.org/en/6.2.x/capture.html#accessing-captured-output-from-a-test-function)

It's nice that we're calling this functions, but we also want to make sure that the output looks like what we expect it to.